Automattic/woocommerce-payments
# Add a task for setting up live payments on WooCommerce Home

Fixes #7736

#### Changes proposed in this Pull Request

Add a task on the WooCommerce home page to remind accounts operating in sandbox mode to set up live payments. It's not necessary to show the task on the Payments Overview page since we already have a notice to prompt a sandbox-mode account to go live.

![321952076-cbe5c04d-b999-462c-b92d-7eb88be9b7f4](https://github.com/Automattic/woocommerce-payments/assets/1288616/9a305e69-5da9-4def-9417-86524d863882)


#### Testing instructions

* Make sure your store is in sandbox mode (formerly called dev mode). By default locally set up stores and JN sites will be created in sandbox mode.
* Visit the WooCommerce Home page and verify that you see the "Set up real payments..." task.
* Clicking the task should open the Set Up Live Payments modal, the same one that's triggered from the sandbox mode notice on top of the Overview and Settings pages.
* Visit the Payments Overview page and verify that the new task is not shown.
* On the same page, click the "Set up payments" button in the sandbox mode notice to ensure that the modal is triggered as expected.
* Link your site to a live account and verify that the "Set up real payments..." task is no longer visible on WooCommerce Home.

-------------------


**Post merge**
